### ISSUE TITLE
Release 13.0.0

### ISSUE THREAD

**Issue by brendandburns** [kubernetes-client/java#1771]
It's time for another release.

cc @tony-clarke-amdocs @yue9944882

**Comment by brendandburns** [comment]
@yue9944882 wdyt?

**Comment by yue9944882** [comment]
@brendandburns definitely SGTM! can you publish the release this time b/c it's a major release, i will take a stab at the release process probably next minor release (13.0.1 or else) in case of any failure?

**Comment by brendandburns** [comment]
Ok, that works for me. I will double check the instructions on a clean machine to validate that they are correct.

**Comment by ydmao** [comment]
hi @brendanburns , really looking forward to 13.0.0 release (because we are blocked by bugs in 12.0.1)! Any timeline on when 13.0.0 release date? thanks!

**Comment by brendandburns** [comment]
Apologies for the delay. I will try to get it done by the end of this week (7/16) at the latest.

**Comment by brendandburns** [comment]
(and what bugs in 12.0.1 we should consider cherry-picking them)

**Comment by ydmao** [comment]
awesome! for us, this one is a must in 13.0.0 - https://github.com/kubernetes-client/java/issues/1721

**Comment by brendandburns** [comment]
I pushed 13.0.0 up to Sonatype and it should be in maven central in ~24 hours.

Closing this issue.

@ydmao if you want me to cherry-pick that PR into 12.0.2 please open a separate issue and we'll get it done.

**Comment by ydmao** [comment]
seems that PR 1721 is already in 13.0.0. We will try using 13.0.0 then. thank you!
